mopidy/internal/path.py

```diff
@@ -196,23 +196,23 @@ def find_mtimes(root, follow=False):
     return mtimes, errors
 
 
-def check_file_path_is_inside_base_dir(file_path, base_path):
-    assert not file_path.endswith(os.sep), (
-        'File path %s cannot end with a path separator' % file_path)
-
+def is_local_path_inside_base_dir(local_path, base_path):
```

Comment:
Right. I wanted to express that path could be dir or file. I could just leave it at file_path, yet i think it is a bit confusing when a file_path is a directory path or is it not?